components/compliance-service/ingest/events/compliance/helper_methods.go

```diff
@@ -41,7 +40,7 @@ func ProfileControlSummary(profile *inspec_api.Profile) *reportingTypes.NodeCont
 	summary := reportingTypes.NodeControlSummary{}
 	for _, control := range profile.Controls {
 		summary.Total++
-		if control.WaiverData != nil && !reflect.DeepEqual(*control.WaiverData, inspec_api.WaiverData{}) && !strings.HasPrefix(control.WaiverData.Message, "Waiver expired") {
+		if control.WaiverData != nil && !pcmp.DeepEqual(*control.WaiverData, inspec_api.WaiverData{}) && !strings.HasPrefix(control.WaiverData.Message, "Waiver expired") {
```

Comment:
see lib/pcmp for the implementation but basically it should be the same as `reflect.DeepEqual` but ignores the internal/private protobuf fields (that used to be `XXX_foo`)